src/LodeRunner.API/src/Controllers/LoadTestConfigsController.cs

```diff
@@ -1,37 +1,76 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+using System;
+using System.Collections.Generic;
 using System.Net;
 using System.Threading;
 using System.Threading.Tasks;
 using AutoMapper;
+using LodeRunner.API.Interfaces;
 using LodeRunner.API.Middleware;
 using LodeRunner.Core.Models;
 using LodeRunner.Data.Interfaces;
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.Cosmos;
 using Swashbuckle.AspNetCore.Annotations;
 
 namespace LodeRunner.API.Controllers
 {
     /// <summary>
-    /// Handle all of the /api/loadtestconfig requests.
+    /// Handle all of the /api/loadtestconfigs requests.
     /// </summary>
     [Route("api/[controller]")]
     [ApiController]
     [SwaggerTag("Create, read, update LoadTest Configurations")]
-    public class LoadTestConfigController : Controller
+    public class LoadTestConfigsController : Controller
     {
+        private const string CosmosDataRequest = "Cosmos data request.";
+        private const string DataNotFound = "Requested data not found in Cosmos.";
+
+        private readonly NgsaLog logger = new ()
+        {
+            Name = typeof(LoadTestConfigsController).FullName,
+            ErrorMessage = "CosmosException",
+            NotFoundError = "Values from Cosmos not found",
+        };
+
         private readonly IMapper autoMapper;
 
         /// <summary>
-        /// Initializes a new instance of the <see cref="LoadTestConfigController"/> class.
+        /// Initializes a new instance of the <see cref="LoadTestConfigsController"/> class.
         /// </summary>
         /// <param name="mapper">The mapper.</param>
-        public LoadTestConfigController(IMapper mapper)
+        public LoadTestConfigsController(IMapper mapper)
         {
             this.autoMapper = mapper;
         }
 
+        /// <summary>
+        /// Returns a JSON array of LoadTestConfig objects.
+        /// </summary>
+        /// <param name="loadTestConfigService">The loadTestConfig service for LRAPI</param>
+        /// <param name="cancellationTokenSource">The cancellation Token Source.</param>
+        /// <returns>IActionResult.</returns>
+        [HttpGet]
+        [SwaggerResponse((int)HttpStatusCode.OK, "Array of `LoadTestConfig` documents or empty array if not found.", typeof(LoadTestConfig[]), "application/json")]
+        [SwaggerResponse((int)HttpStatusCode.NoContent, "`Data not found.`", null, "text/plain")]
```

Comment:
Please add a check for lodeTestconfigs to determine if it is an empty array and then return HttpStatusCode.NoContent instead of HttpStatusCode.OK